languages/ruby/outline.scm

```diff
@@ -19,6 +19,12 @@
     "module" @context
     name: (_) @name) @item
 
+
+(assignment
+    (((constant) @constant
+    (#match? @constant "^[A-Z\\d_]")) @name
```

Comment:
🤔 Why match on digits and underscores here? They're valid inside the constant name, but not at the beginning.